pmm/aws-staging-start.groovy

```diff
@@ -201,6 +206,7 @@ pipeline {
                                 ],
                                 "EbsOptimized": false,
                                 "ImageId": "ami-15e9c770",
```

Comment:
As I told earlier it is very old amazon linux image based on centos 6,  maybe if you update it to  amazon linux 2 which is based on centos 7 a lot of your issues will be fixed. Anyway you need to do it sooner or later.

Reply:
Good point Slava, but we need this pipeline working. So we will have to address this upgrade in a separate PR, which will be dedicated to migration and maybe some refactoring thereof.

Reply:
@hors Which image would be the best replacement for `ami-15e9c770`?

Reply:
@atymchuk try to find Amazon Linux 2 suitable for your jenkins (I mean in your region) using https://us-east-2.console.aws.amazon.com/ec2/v2/home?region=us-east-2#LaunchInstanceWizard e.g. this one ami-09558250a3419e7d0